bql/semantic/expression_test.go

```diff
@@ -96,10 +96,10 @@ func TestEvaluationNode(t *testing.T) {
 	for _, entry := range testTable {
 		got, err := entry.eval.Evaluate(entry.r)
 		if !entry.err && err != nil {
-			t.Errorf("failed to evaluate op %q for %v on row %v with error: %v", entry.eval.(*evaluationNode).op.String(), entry.eval, entry.r, err)
+			t.Errorf("failed to evaluate op %q for %v on row %v with error: %v", entry.eval.(*evaluationNode).operation.String(), entry.eval, entry.r, err)
```

Comment:
should be ok to remove ".String()"

When an operand implements the method String, the method will be invoked to convert the object to a string.

Reply:
Well pointed. Thank you Jose, I have just done so.